src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -1,42 +1,39 @@
 package edu.kis.vh.nursery;
 
-public class DefaultCountingOutRhymer {
-	public static final int NUMBERS_ARRAY_SIZE = 12;
-	public static final int INITIAL = -1;
-	public static final int RETURN = -1;
+import edu.kis.vh.nursery.structures.IntArrayStack;
 
+public class DefaultCountingOutRhymer {
+	private IntArrayStack arrStack;
```

Comment:
The goal of the lab was to make using IntLinkedList and IntArrayStack interchangeable. You should use the interface here.